ui/guest/src/react/GuestProxy.tsx

```diff
@@ -180,13 +179,17 @@ export function GuestProxy() {
       }
     };
 
-    const insertElement = ($element: JQuery<any>, $daddy: JQuery<any>, targetIndex: string | number): void => {
+    const insertElement = (
+      element: Element | HTMLElement,
+      daddy: Element | HTMLElement,
+      targetIndex: string | number
+    ): void => {
       const index = typeof targetIndex === 'string' ? parseInt(popPiece(targetIndex)) : targetIndex;
-      const $siblings = $daddy.find('> *');
-      if ($siblings.length === index) {
-        $daddy.append($element);
+      const siblings = daddy.querySelectorAll('*');
```

Comment:
Not an equivalent selection (missing >). Triple check for similar instances of bad selector translation. 

Here we're looking for direct descendants, so .children might be sufficient and faster. Check.